entities/credential/credential-key-value/query.ts

```
@@ -0,0 +1,46 @@
+      createdAt: true,
+      updatedAt: true,
+    } satisfies Prisma.CredentialKeyValuePairSelect
+  }
+
+  static getSimpleInclude() {
+    return {} satisfies Prisma.CredentialKeyValuePairInclude
+  }
+
+  static getInclude() {
+    return {
+      ...this.getSimpleInclude(),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)